src/app/containers/Include/index.jsx

```diff
@@ -13,6 +14,8 @@ import VjAmp from './amp/VjAmp';
 
 const logger = nodeLogger(__filename);
 
+const FALLBACK_CLASSIFICATIONS = ['vj-amp-not-supported', 'idt1-amp'];
```

Comment:
I think the include business rules have bled into the frontend container from the 'backend pre-processor rules here, principally the concept of classifications.

I'd favour the block data being given a `showFallback` prop rather than exposing the classifications to the frontend. https://github.com/bbc/simorgh/pull/7180/files#diff-ace5f927a44cc0bc96a151479cba4bbaR120 

That way if we change the `includeClassifier` or indeed to include converter, we are less likely to have to make changes in this frontend container, decreasing coupling,